NSTTutorials/ResponseLoggerInjections/README.md

```diff
@@ -0,0 +1,45 @@
+[_Return to Tutorials Overview_](../README.md)
+
+# Response Logger Injections
+
+### Overview
+
+For use cases where the mock response files / data needs to be modified in some way before the file is generated, we can leverage response loggers.
+
+### Topics Covered
+
+1. [Modifying response data](#modifying-response-data)
+
+### References
+- [ResponseLoggerInjector](../../NST/src/main/java/com/ebay/service/logger/injection/ResponseLoggerInjector.java)
+
+## Modifying response data
+
+An example use case for when we would want to modify response data using the ResposneLoggerInjector could be some scenario in which you would want the fully validated mock response, but then wanted to modify some specific field’s **VALUE** to something else, for testing purposes.
+
+This is accomplished at the service wrapper level. Service wrappers have a method, `getResponseLoggerInjector`, that requires an implementation of *`ResponseLoggerInjector`.* 
+
+*`ResponseLoggerInjector`* only has 1 method, `processServiceResponse` , which is where we will be making our modifications to the response data. 
```

Comment:
Make clear that the modification will only be to the mock data written to file. The response model from the service does NOT adopt the injection changes.

Reply:
Done in 3d43f0926bdcdf8d1cddbe7d5466662922cf51bb